README.md

```diff
@@ -83,12 +73,41 @@ The [docker/](docker/) directory contains sample files related to container-base
 The Sling Starter will execute two suites of tests using the `maven-surefire-plugin`:
 
 1. A small set of smoke tests, embedded in the project, that verify the basic functionality of the Starter
-1. An extensive set of end-to-end tests that verify the overall functionality of the Starter and the bundles that are embedded into it
+2. An extensive set of end-to-end tests that verify the overall functionality of the Starter and the bundles that are embedded into it
 
 By default, these are both executed when building the project against an Oak SegmentNodeStore backend.
 
 Additionally, when the `ci` profile is enabled the smoke tests are also executed in against an Oak DocumentNodeStore backend. For technical resons, the full end-to-end tests are not executed.
 
+
+## Docker
+
+> The Sling Starter Docker image is only for testing/learning purposes. For your custom applications, 
+> please configure your [Feature model](https://github.com/apache/sling-org-apache-sling-feature) and 
+> build your image with all dependencies and configurations built in.
+
+Our example Docker image uses `/opt/sling/bin/launch.sh` script as the `entrypoint`. It supports the 
+following environment variables:
+
+| Environment variable | Description                                                          | Default  |
+|----------------------|----------------------------------------------------------------------|----------|
+| `SLING_DEBUG`        | Run Sling Starter in debug mode. Allowed values are `false`, `true`. | `false`  |
+| `JAVA_DEBUG_PORT`    | Custom debug port.                                                   | `5005`   |
```

Comment:
I think it's simpler if we enable debug whenever `JAVA_DEBUG_PORT` is defined and do away with `SLING_DEBUG`.

Reply:
done